keyboards/7skb/rev1/config.h

```diff
@@ -21,7 +21,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xFEED
 #define PRODUCT_ID      0x3060
-#define DEVICE_VER      0x0007
+#define DEVICE_VER      0x36e1
```

Comment:
```suggestion
#define DEVICE_VER      0x0007
```

`DEVICE_VER` is a BCD version number (eg. `0x0123` = 1.23).